docs/config-file-package-configuration.yml.md

```diff
@@ -0,0 +1,101 @@
+# The package configuration file (Experimental)
+
+A package configuration sets up path excludes and license finding curations specific to a package (dependency) and 
+provenance. It is thus a similar concept to the `.ort.yml` file which allows to do similar configuration for projects as
+opposed to packages.
+
+## Specifying the target package and provenance
+
+Each package configuration applies exactly to one package-Id and provenance which must be specified. The provenance
+can either specify a *source artifact* or a *VCS location and revision*. Thus in case one scans the *source artifact*
+and the *VCS* for each package, then two package configurations would be required in order to address issues with both
+scans. For example configurations for `ansi-styles 4.2.1` would specify the following... 
+  
+* to configure the VCS scan:
+```yaml
+  id: "NPM::ansi-styles:4.2.1"
+  vcs:
+    type: "Git"
+    url: "https://github.com/chalk/ansi-styles.git"
+    revision: "74d421cf32342ac6ec7b507bd903a9e1105f74d7"
+```
+* to configure the source artifact scan:
+```yaml
+  id: "Maven:org.ossreviewtoolkit.ort:1.2.3"
```

Comment:
You say `example configurations for `ansi-styles 4.2.1`` but the source artifact scan is about `Maven:org.ossreviewtoolkit.ort:1.2.3` and `ansi-styles-3.2.1.tgz` - is this a bug?